index.js

```diff
@@ -1,29 +1,74 @@
-/*NOTE: the size of the files extracted from the <input type="file"> element
-            is measured in Bytes. Thus, all convertions from Bytes, to KiloBytes, MegaBytes 
-            and so on must take that into account. */
-
 const KB = 1024;
 const MB = KB*KB;
 const GB = KB*MB;
 
+const supported_extensions = ["jpg",
+    "jpeg","png","gif","bmp","tiff",
```

Comment:
For constants please follow these naming convention : 

`const SUPPORTED_EXTENTIONS = [.....]`